api/client/samples/similar_regions_Frechet/metric.py

```diff
@@ -0,0 +1,179 @@
+### This file describes the metric to be used when doing region similarity computation.
+
+# metric_properties mostly gives information needed for data download.
+# In it, "api_data" are API parameters (metric_id, item_id, etc.) needed to uniquely identify data for Gro API.
+#
+# For time series, it is recommended to provide start_date set to the start of calendar year (XXXX-01-01)
+# and end_date set to the end of year (YYYY-12-31) fully within data availability for particular item.
+# Otherwise data might become biased towards particular part of the year.
+# This is especially important when searching is expected to include regions in both Northern and Southern hemispheres.
+#
+# "properties" are:
+#    type (required) : can be timeseries or pit (point-in-time)
+#    norm (optional) : normalization constant. Data is DIVIDED by it. If not given, each value will be divided by this item's data std.
+# It is recommended to provide normalization constants, ideally the std of corresponding item over the entire world
+# or other region of primary interest. Otherwise data will be re-normalized differently for runs with different search regions,
+# which might result in counter-intuitive reordering of similar regions.
+
+# metric_instance is a item:weight dictionary specifying the particular collection of items and their weights
+# to be used in contructig similar_region object. It should be a subset of metric_properties
+# Each data point belonging to an item will be MULTIPLIED by sqrt of the weight
+
+# Full calculation before applying the distance measure is therefore sqrt(weight)*data/norm.
+# Note that taking sqrt of user-provided weights here assumes user wants something like
+# dist^2 = \sum w_user_i*(x_i - y_i)^2 (note non-squared w_user_i here)
+# which is usual specification in weighted L2 norm (least squares)
+
+
+# Just an example. This is intended to be define in the user code
+metric_instance = {
```

Comment:
should we include weight in the variable name?

Reply:
Probably not, at least not yet. While it is true that right now this only contains property weights, I actually moved things back and forth between metric_instance and metric_description and still not sure what the best split is. For example, "norm" might be better suited for metric_instance...

Reply:
renamed to metric_weights